vdev/src/testing/runner.rs

```diff
@@ -21,6 +22,35 @@ const TEST_COMMAND: &[&str] = &[
     "--no-fail-fast",
     "--no-default-features",
 ];
+// The upstream container we publish artifacts to on a successful master build.
+const UPSTREAM_IMAGE: &str =
+    "docker.io/timberio/vector-dev:sha-3eadc96742a33754a5859203b58249f6a806972a";
+
+static CONTAINER_TOOL: Lazy<OsString> =
+    Lazy::new(|| env::var_os("CONTAINER_TOOL").unwrap_or_else(detect_container_tool));
+
+fn detect_container_tool() -> OsString {
+    for tool in ["docker", "podman"] {
```

Comment:
Wheres my support for [krunvm](https://github.com/containers/krunvm)! /s

Reply:
For now there is a requirement that the container tool uses the `docker` comand-line interface. If `krunvm` can do that, then I'll add it. Otherwise, it'll take more work to support.